test/CMakeLists.txt

```diff
@@ -118,9 +118,10 @@ function(make_test)
           CXX_STANDARD 11)
     endif(${mt_args_CXX11})
 
-	  set_target_properties(${target}
-      PROPERTIES
-        FOLDER "Tests")
+    set_target_properties(${target} PROPERTIES FOLDER "Tests")
```

Comment:
You should combine these into one set_target_properties

Reply:
Addressed